### PR TITLE
Pr mesh report

### PR DESCRIPTION
**Pull request by crisrogo**
Dijiste al chile pero no deja :(

### CODE REVIEW

File: imatools/utils.py
Comment (alonsoJASL) [inline imatools/utils.py:1]: This should be in the common folder. I will move things around, push to this PR and then accept the changes.

**Review by alonsoJASL — CHANGES_REQUESTED** [review]
I will move the utils.py to the common folder and then move on with the PR

File: imatools/utils.py
Comment (crisrogo) [reply imatools/utils.py:1]: There is a chance that those are duplicated functions you already had coded (read_elem etc).

Comment (alonsoJASL) [reply imatools/utils.py:1]: Yes, adjustments were made and tested.

**Review by alonsoJASL — APPROVED** [review]
All the changes were implemented. An overview: 
+ Moved `utils.py` to `common` folder 
+ Moved some I/O functions to improve on existing in `common.ioutils` 
+ Moved generic plotting functions to `common.plotutils` 
+ Changed some inputs and added parser groups to make running `python imatools/mesh_report.py -h` easier to read

**Pull request merged**